map_view/display-drawing-status/src/main/java/com/esri/samples/display_drawing_status/DisplayDrawingStatusSample.java

```diff
@@ -55,21 +56,25 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
       // create progress bar
       ProgressBar progressBar = new ProgressBar();
       progressBar.setMaxWidth(240.0);
 
-      // create a ArcGISMap with topographic basemap
-      final ArcGISMap map = new ArcGISMap(Basemap.createTopographic());
+      // create a map with the topographic basemap style
+      final ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_TOPOGRAPHIC);
 
-      // create a starting viewpoint for the ArcGISMap
+      // create a starting viewpoint for the map
       SpatialReference spatialReference = SpatialReferences.getWebMercator();
       Point bottomLeftPoint = new Point(-1.5054808160556655E7, 2718975.702666207, spatialReference);
       Point topRightPoint = new Point(-6810317.90634398, 6850505.377826911, spatialReference);
       Envelope envelope = new Envelope(bottomLeftPoint, topRightPoint);
       Viewpoint viewpoint = new Viewpoint(envelope);
 
-      // set the initial viewpoint of ArcGISMap to viewpoint
+      // set the initial viewpoint of the map
```

Comment:
was there a reason not to use `mapView.setViewpoint` here?

Reply:
@Rachael-E think it just got missed, thanks! I've amended this and slightly reduced the number of separate lines of constructors. See what you think?